__tests__/commands/_helpers.js

```diff
@@ -28,6 +28,18 @@ export const runInstall = run.bind(
   [],
 );
 
+export const runInstallWithoutCheck = run.bind(
```

Comment:
Oh I see. So just removing the folder from `node_modules` isn't enough and will cause check to fail since the module isn't installed.